config/repo-values.yaml

```diff
@@ -64,3 +64,5 @@ sync:
       url: https://ceph.github.io/csi-charts
     - name: reactiveops-stable
       url: https://charts.reactiveops.com/stable
+    - name: yourls
+      url: https://charts.yourls.org
```

Comment:
Thanks @LeoColomb 

FYI, this link is not working

Reply:
https://charts.yourls.org/index.yaml 🙂

Reply:
oops

Thanks for keeping on this for 6 months!